### PR TITLE
Add Integration tests for Operation Policy Import/Export APIs

### PR DESCRIPTION
**Pull request by BLasan**
**Purpose**
Fixes: [wso2/api-manager#223](https://github.com/wso2/api-manager/issues/223) [wso2/api-manager#489](https://github.com/wso2/api-manager/issues/489)

**Approach**
Implemented the following test cases.
- testCommonOperationPolicyExport
- testInvalidCommonOperationPolicyExport
- testImportNewCommonOperationPolicy
- testImportExistingCommonOperationPolicy
- testImportInvalidCommonOperationPolicy
- testCommonOperationPolicyExportWithJSONContent

### CODE REVIEW

File: modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIPublisherImpl.java [hunk 1]
```diff
@@ -30,29 +31,7 @@
...
-import org.wso2.am.integration.clients.publisher.api.v1.GraphQlSchemaApi;
-import org.wso2.am.integration.clients.publisher.api.v1.GraphQlSchemaIndividualApi;
-import org.wso2.am.integration.clients.publisher.api.v1.OperationPoliciesApi;
-import org.wso2.am.integration.clients.publisher.api.v1.RolesApi;
-import org.wso2.am.integration.clients.publisher.api.v1.ScopesApi;
-import org.wso2.am.integration.clients.publisher.api.v1.SubscriptionsApi;
-import org.wso2.am.integration.clients.publisher.api.v1.ThrottlingPoliciesApi;
-import org.wso2.am.integration.clients.publisher.api.v1.UnifiedSearchApi;
-import org.wso2.am.integration.clients.publisher.api.v1.ValidationApi;
+import org.wso2.am.integration.clients.publisher.api.v1.*;
```
Comment (wasuradananjith) [inline modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIPublisherImpl.java:34]: Shall we revert this wildcard import?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java [hunk 2]
```diff
@@ -19,39 +19,69 @@
...
 import org.wso2.am.integration.tests.api.lifecycle.APIManagerLifecycleBaseTest;
+import org.wso2.andes.util.FileUtils;
+import org.wso2.carbon.apimgt.api.model.OperationPolicyData;
 import org.wso2.carbon.automation.test.utils.http.client.HttpResponse;
 import org.apache.http.client.HttpClient;
+import wiremock.com.google.common.io.Files;
 
-import java.io.File;
-import java.io.IOException;
+import java.io.*;
```
Comment (wasuradananjith) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java:71]: A wildcard import. Let's remove this?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java [hunk 3]
```diff
@@ -68,12 +98,26 @@ public class OperationPolicyTestCase extends APIManagerLifecycleBaseTest {
...
     private String applicationId;
     private String apiId;
     private String newVersionAPIId;
     private String accessToken;
     private Map<String, String> policyMap;
 
+    private File exportedOperationPolicyZip;
+    private String exportUrl;
+    private RestAPIPublisherImpl restAPIPublisherExport;
+//    private String publisherURLHttps;
```
Comment (wasuradananjith) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java:119]: Do we need this?

File: modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIPublisherImpl.java (hunk 1 above)
Comment (BLasan) [reply modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIPublisherImpl.java]: Fixed by [669abe4](https://github.com/wso2/product-apim/pull/12938/commits/669abe4beafa46b92f046a94eb15766805cd74fa)

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java (hunk 2 above)
Comment (BLasan) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java]: Fixed by [669abe4](https://github.com/wso2/product-apim/pull/12938/commits/669abe4beafa46b92f046a94eb15766805cd74fa)

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java (hunk 3 above)
Comment (BLasan) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java]: Fixed by [669abe4](https://github.com/wso2/product-apim/pull/12938/commits/669abe4beafa46b92f046a94eb15766805cd74fa)

File: modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIPublisherImpl.java (hunk 1 above)
Comment (BLasan) [reply modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIPublisherImpl.java]: Fixed by [a7ae252](https://github.com/wso2/product-apim/pull/12938/commits/a7ae2523e0b98b3d6c24c4c351b07dae9290fa3d)

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java (hunk 2 above)
Comment (BLasan) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java]: Fixed by [a7ae252](https://github.com/wso2/product-apim/pull/12938/commits/a7ae2523e0b98b3d6c24c4c351b07dae9290fa3d)

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java (hunk 3 above)
Comment (BLasan) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java]: Fixed by [a7ae252](https://github.com/wso2/product-apim/pull/12938/commits/a7ae2523e0b98b3d6c24c4c351b07dae9290fa3d)

**Review by wasuradananjith — APPROVED** [review]

**Pull request merged**